lgc/patch/PatchSetupTargetFeatures.cpp

```diff
@@ -187,6 +187,10 @@ void PatchSetupTargetFeatures::setupTargetFeatures(Module *module) {
 
     builder.addAttribute("target-features", targetFeatures);
 
+    // Prevent spilling of VGPRs holding SGPR spills as this can have undefined behaviour in callee functions.
```

Comment:
Thanks for fixing this. Personally I think it maybe helpful to better comment the problem. The problem is the current way to allocate WWM-VGPR used for SGPR spilling during RA is broken when the WWM-VGPR is either spilled or its live-range was split. So we choose to pre-allocate those WWM-VGPRs for SGPR spills. It maybe more proper to say we will re-visit the problem when the backend issue is fixed. I agree SetupTargetFeature is the reasonable place for this attribute.

Reply:
I choose to not describe the exact failure mode of the backend in this comment for brevity.
I felt "undefined behaviour" a sufficiently accurate a description as on a good day it'll all work fine, but on a bad day you get lane clobbering and unexpected behaviour.
The comment on the next line already basically says "remove this later".
If this wasn't clear then I'll reword the comment.